de.metas.report/metasfresh-report-via-ssh-tunnel.launch

```diff
@@ -7,12 +7,10 @@
 <booleanAttribute key="org.eclipse.jdt.launching.ALLOW_TERMINATE" value="false"/>
 <mapAttribute key="org.eclipse.jdt.launching.CONNECT_MAP">
 <mapEntry key="hostname" value="localhost"/>
-<mapEntry key="port" value="18791"/>
+<mapEntry key="port" value="8791"/>
```

Comment:
pls stick with port `18791` because it's a common practice to mal `localhost:1xxxx` to `remote:xxxx`